### PR TITLE
Milestone2

### PR DESCRIPTION
**Pull request by Tahirbhalli**
create-react project
 > Add Display component
 > Add buttonpannel component
 > Add button component
deploy to heroku
To Run locally:
npm install
npm start
note: I was not able to submit my project milestones So in Heroku I deployed the whole project, and Style will be in milestone 3

### CODE REVIEW

File: src/component/Button.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/button-has-type */
+import React from 'react';
+
+export default function Button(props) {
+  return (
+    <button className="">{props.name}</button>
+  );
+}
```
Comment (shloch) [inline src/component/Button.js]: `name` prop should be a string, enforce this with `propTypes` [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/button/validateAsString.png)]

File: src/component/Display.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+/* eslint-disable react/destructuring-assignment */
+import React from 'react';
+
+export default function Display(props) {
+  return (
+    <div>
+      <input placeholder="enter the number" value={props.value} />
+    </div>
+  );
+}
```
Comment (shloch) [inline src/component/Display.js]: - `value` props should be called `"result"` instead according to requirement, [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/incomingProps.png)]
- it should also be a string, re-inforce this with `propTypes` [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/validateAsString.png)]

- this prop's default value should be `'0'`, confirm this with `defaultProps` [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/DefaultProp.png)]

- Also, why should it be inside an `input`, it's a `DISPLAY` component, it should just display the value of this prop at the moment

**Review by shloch — CHANGES_REQUESTED** [review]
## Hello @Tahirbhalli 

Great job with the implementations, it's really good

We just have to enforce props with `propTypes` and `defaultProps` (when necessary)

Let's just work on this

Happy coding

File: src/component/Display.js (hunk 2 above)
Comment (Tahirbhalli) [reply src/component/Display.js]: the last point will be updated in milestone 4 or 5 when I will use the event handles

**Review by shloch — APPROVED** [review]
## Hello @Tahirbhalli , 

accomplished

Great job with the additions, I could see other additions like `color` props, which is introduced into `milestone 3`....

Keep up with the good work

Happy Coding

**Pull request closed without merging**